upb/bindings/lua/msg.c

```diff
@@ -91,11 +91,12 @@
  *            lupb_Arena                +-weak-|cache|-weak-+     lupb_Arena
  *                 |  ^                 |      +-----+      |        ^   |
  *                 |  |                 V                   V        |   |
- * Lua level       |  +------------lupb_msg              lupb_msg----+   |
+ * Lua level       |  +------------lupb_Message              lupb_Message----+ |
  * ----------------|-----------------|-------------------------|---------|------
  * upb level       |                 |                         |         |
  *                 |            +----V----+               +----V----+    V
- *                 +->upb_Arena | upb_msg |               | upb_msg | upb_Arena
+ *                 +->upb_Arena | upb_Message |               | upb_Message |
+ * upb_Arena
  *                              +------|--+               +--^------+
  *                                     +---------------------+
```

Comment:
you ascii art got ugly.  Might want to hit it again

Reply:
Done.